llvm/test/MC/Disassembler/AMDGPU/gfx11_dasm_vop1_dpp8.txt

```diff
@@ -253,30 +253,60 @@
 # GFX11: v_mov_b32_dpp v255, v255 dpp8:[0,0,0,0,0,0,0,0] fi:1 ; encoding: [0xea,0x02,0xfe,0x7f,0xff,0x00,0x00,0x00]
 0xea,0x02,0xfe,0x7f,0xff,0x00,0x00,0x00
 
+# GFX11: v_mov_b32_e64_dpp v5, -v1 dpp8:[7,6,5,4,3,2,1,0] ; encoding: [0x05,0x00,0x81,0xd5,0xe9,0x00,0x00,0x20,0x01,0x77,0x39,0x05]
```

Comment:
This, and other new cases below are VOP3. This should be in gfx11_dasm_vop3_dpp8_from_vop1.txt